tools/pythonpkg/duckdb-stubs/__init__.pyi

```diff
@@ -203,7 +204,7 @@ class DuckDBPyRelation:
     @property
     def dtypes(self) -> list: ...
     @property
-    def description(self) -> object: ...
+    def description(self) -> list: ...
```

Comment:
Why is descriprion a list here but an Optional[list] in other stubs?

Reply:
This confused me as well!

description is only None if there's no result set available - so the description on the DuckDBPyRelation/Result is never None